qmetaobject_impl/src/qobject_impl.rs

```diff
@@ -875,7 +875,12 @@ pub fn generate(input: TokenStream, is_qobject: bool, qt_version: QtVersion) ->
             .map(|arg| {
                 let n = &arg.name;
                 let ty = &arg.typ;
-                quote! { unsafe { ::std::mem::transmute::<& #ty, *mut ::std::os::raw::c_void>(& #n) } }
+                quote! {
+                    {
+                        let mut n = #n;
+                        (&mut n as *mut #ty as *mut ::std::os::raw::c_void)
```

Comment:
that doesn't look right. It seems it's taking the address of `n` on the stack.

Reply:
I'm not sure either. This should be catched by the compiler though. The code I've tested did not generate any errors. The `let` binding was necessary to get a mutable reference `&mut`. The compiler suggests to annotate the function argument with `mut`. I'll check if that is possible.

Reply:
In this case it might not be catched by the compiler since the user of the pointer uses unsafe code.

Reply:
Good point. I now use marked the function argument as `mut` so that a `&mut` reference can be taken.